Keyboard/DefaultKeyboard.swift

```diff
@@ -39,15 +39,7 @@ func defaultKeyboard() -> Keyboard {
     keyModeChangeNumbers.toMode = 1
     defaultKeyboard.add(key: keyModeChangeNumbers, row: 3, page: 0)
     
-    if #available(iOSApplicationExtension 11.0, *) {
-        if (KeyboardViewController.shared?.needsInputModeSwitchKey == true) {
-            let keyboardChange = Key(.keyboardChange)
-            defaultKeyboard.add(key: Key(keyboardChange), row: 3, page: 0)
-            defaultKeyboard.add(key: Key(keyboardChange), row: 3, page: 1)
-            defaultKeyboard.add(key: Key(keyboardChange), row: 3, page: 2)
-        } else if (KeyboardViewController.shared?.needsInputModeSwitchKey == false) {
```

Comment:
I removed the else if because it does not do anything.

Reply:
Agreed